src/quickgui/plugin/editor/qgsquickexternalresource.qml

```diff
@@ -39,36 +43,57 @@ Item {
     height: window.height
     width: window.width
     edge: Qt.RightEdge
+    imageButtonSize: button.height
   }
 
   Image {
     property var currentValue: value
 
     id: image
-    width: 200 * QgsQuick.Utils.dp
+    width: customStyle.height * 3
     autoTransform: true
     fillMode: Image.PreserveAspectFit
+    visible: currentValue
 
     Component.onCompleted: image.source = getSource()
 
     function getSource() {
       if (image.status === Image.Error)
-        return QgsQuick.Utils.getThemeIcon("ic_broken_image_black")
+        return ""
```

Comment:
should it be 

if (image.currentValue ...)
else 
return ""

?

Also now you have separate image for broken/missing and separate one for working version? Why is that?

Reply:
Reverted back to one image for icons/preview photo